While testing on e107_sh042 in layout task I had this error (which caused the build to fail, therefore my built scene is incomplete):
```
Traceback (most recent call last):

  File "/home/sharkitty/work/normaal/repos/openpype-for-normaal/openpype/hosts/blender/api/ops.py", line 129, in execute
    result = callback(*args, **kwargs)

  File "/home/sharkitty/work/normaal/repos/openpype-for-normaal/openpype/hosts/blender/api/ops.py", line 1051, in _build_first_workfile
    build_workfile()

  File "/home/sharkitty/work/normaal/repos/openpype-for-normaal/openpype/hosts/blender/scripts/build_workfile.py", line 758, in build_workfile
    build_layout(project_name, asset_name)

  File "/home/sharkitty/work/normaal/repos/openpype-for-normaal/openpype/hosts/blender/scripts/build_workfile.py", line 508, in build_layout
    load_subset(project_name, board_repre, "Background")

  File "/home/sharkitty/work/normaal/repos/openpype-for-normaal/openpype/hosts/blender/scripts/build_workfile.py", line 175, in load_subset
    return load_container(loader, representation)

  File "/home/sharkitty/work/normaal/repos/openpype-for-normaal/openpype/pipeline/load/utils.py", line 384, in load_container
    return load_with_repre_context(

  File "/home/sharkitty/work/normaal/repos/openpype-for-normaal/openpype/pipeline/load/utils.py", line 308, in load_with_repre_context
    return loader.load(repre_context, name, namespace, options)

  File "/home/sharkitty/work/normaal/repos/openpype-for-normaal/openpype/hosts/blender/plugins/load/load_image.py", line 113, in load
    container, datablocks = super().load(*args, **kwargs)

  File "/home/sharkitty/work/normaal/repos/openpype-for-normaal/openpype/hosts/blender/api/plugin.py", line 384, in wrapper_exec_process
    return process_func(*args, **kwargs)

  File "/home/sharkitty/work/normaal/repos/openpype-for-normaal/openpype/hosts/blender/api/plugin.py", line 1051, in load
    assert Path(self.fname).exists(), f"{self.fname} doesn't exist."

AssertionError: /home/sharkitty/work/normaal/openpype/ww/main/_RENDERS/Shots/e107/e107_sq009/e107_sq009_e107_sh042/review/BoardReference/v001/woollywoolly_e107_sq009_e107_sh042_BoardReference_v001.mov doesn't exist.
```
I ran this test in SFTP remote on Fedora-KDE 38.

I'm not entirely sure if I should consider that relevant to your PR. One one hand, I think the build fails because of an SFTP issue (which might already be fixed in another branch). But on the other hand, if I understand your PR correctly, maybe it should just throw a warning and continue to build what can be built in this case. What do you think about it?

--- a/openpype/hosts/blender/scripts/build_workfile.py
+++ b/openpype/hosts/blender/scripts/build_workfile.py
@@ -297,7 +297,8 @@ def load_casting(project_name: str, shot_name: str) -> Set[OpenpypeContainer]:
             all_datablocks.update(datablocks)
         except TypeError:
             print(
-                f"Cannot load {representation['context']['asset']} {representation['context']['subset']}."
+                f"Cannot load {representation['context']['asset']}"
+                f"{representation['context']['subset']}."
             )
 
     return containers, all_datablocks
@@ -389,6 +390,7 @@ def build_layout(project_name, asset_name):
 
     # Load casting from kitsu breakdown.
     containers = {}
+    errors = []
     try:
         _casting_containers, casting_datablocks = load_casting(
             project_name, asset_name
@@ -404,14 +406,18 @@ def build_layout(project_name, asset_name):
                 continue
             if c.outliner_entity not in layout_container.children.values():
                 layout_container.children.link(c.outliner_entity)
-            if c.outliner_entity in bpy.context.scene.collection.children.values():
+            if (
+                c.outliner_entity
+                in bpy.context.scene.collection.children.values()
+            ):
                 bpy.context.scene.collection.children.unlink(c.outliner_entity)
 
         # Create GDEFORMER collection
         create_gdeformer_collection(
             layout_instance.datablock_refs[0].datablock
         )
-    except RuntimeError:
+    except RuntimeError as err:
+        errors.append(f"Load casting failed ! {err}")
         # Wait for download
         wait_for_download(project_name, [board_repre, audio_repre])
 
@@ -475,7 +481,8 @@ def build_layout(project_name, asset_name):
                 ),
                 None,
             )
-    except RuntimeError:
+    except RuntimeError as err:
+        errors.append(f"Build setdress failed ! {err}")
         camera_collection = None
 
     # Ensure camera instance
@@ -524,6 +531,8 @@ def build_layout(project_name, asset_name):
             "Background",
         )
 
+    assert not errors, "; ".join(errors)
+
 
 def build_anim(project_name, asset_name):
     """Build anim workfile.
@@ -558,6 +567,7 @@ def build_anim(project_name, asset_name):
     )
 
     # Switch hero containers to versioned
+    errors = []
     for container in bpy.context.scene.openpype_containers:
         container_metadata = container["avalon"]
         # Get version representation
@@ -606,7 +616,10 @@ def build_anim(project_name, asset_name):
                 switch_container(
                     container_metadata, version_representation, loader
                 )
-            except RuntimeError:
+            except RuntimeError as err:
+                errors.append(
+                    f"Switch versioned failed for {container.name}: {err}"
+                )
                 continue
 
     # Substitute overridden GDEFORMER collection by local one
@@ -668,6 +681,8 @@ def build_anim(project_name, asset_name):
 
     # load the board mov as image background linked into the camera
     load_subset(project_name, board_repre, "Background")
+
+    assert not errors, "; ".join(errors)
 
 
 def build_lipsync(project_name: str, shot_name: str):

--- a/openpype/hosts/blender/scripts/build_workfile.py
+++ b/openpype/hosts/blender/scripts/build_workfile.py
@@ -317,7 +317,7 @@ def build_model(project_name, asset_name):
     create_instance("CreateModel", "modelMain", useSelection=True)
     # load the concept reference as image reference in the scene.
     download_and_load_subset(
-        project_name, asset_name, "conceptReference", "Reference"
+        project_name, asset_name, "ConceptReference", "Reference"
     )
 
 

